bibliopixel/drivers/PiWS281X.py

```diff
@@ -0,0 +1,52 @@
+# Original code by msurguy: https://github.com/ManiacalLabs/BiblioPixel/issues/51#issuecomment-228662943
+from . channel_order import ChannelOrder
+from . driver_base import DriverBase
+from .. import log
+from .. import gamma
+try:
+    from neopixel import Adafruit_NeoPixel, Color as NeoColor
+except:
+    log.error("""PiWS281X Requires the rpi_ws281x C extension.
+Install rpi_ws281x with below commands:
+git clone https://github.com/jgarff/rpi_ws281x.git
+cd rpi_ws281x
+sudo apt-get install python-dev swig scons
+sudo scons
+cd python
+sudo python3 setup.py build install
```

Comment:
That last line is dangerous if they are using virtualenvs, as they would likely install it for a version of Python that isn't theirs... but not quite sure how to proceed?

Consider having two lines, one if you are using virtualenv and one if not??